tests/e2e/default/db/ongoing-replication.wdio-spec.js

```diff
@@ -35,12 +33,18 @@ describe('ongoing replication', function() {
   });
 
   after(async () => {
+    await utils.stopSentinel();
     await sentinelUtils.skipToSeq();
-    await sentinelUtils.waitForSentinel();
+    await utils.startSentinel();
   });
 
   afterEach(async () => {
     await browser.throttle('online');
+    const isRevertingSettings = utils.revertSettings(true);
+    if (isRevertingSettings) {
+      await isRevertingSettings;
+      await commonPage.sync(true);
+    }
```

Comment:
It feels like this should be part of the `revertSettings` function. Surely you always need to sync after changing settings?

Reply:
I thought about that too, I'm not entirely sure if it's always the case. 
I'm afraid this will start generating a lot of other e2e tests to flake or fail because we suddenly introduce this new behavior in a PR that attempts to stabilize an e2e test.

Reply:
Yeah it feels very fragile. Fine to leave it as is for now.

Reply:
I think you're totally right to point to this. I was very close to implementing that as well.